src/main/java/org/broadinstitute/hellbender/tools/recalibration/BQSRGatherer.java

```diff
@@ -1,84 +0,0 @@
-package org.broadinstitute.hellbender.tools.recalibration;
-
-import org.apache.commons.collections4.CollectionUtils;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-import org.broadinstitute.hellbender.exceptions.GATKException;
-import org.broadinstitute.hellbender.utils.commandline.Gatherer;
-import org.broadinstitute.hellbender.utils.io.IOUtils;
-import org.broadinstitute.hellbender.utils.report.GATKReport;
-
-import java.io.File;
-import java.io.IOException;
-import java.io.PrintStream;
-import java.util.*;
-
-/**
- * Merges BQSR Recaliration tables.
- */
-public final class BQSRGatherer extends Gatherer {
```

Comment:
It's not actually being used -- the `@Gather` annotation is only used by the GATK3 job runner Queue, which we don't support (yet). If we want to support Queue in the future we can restore this, but we'll need to make a bunch of other changes to our codebase like using `@Input` and `@Output` instead of `@Argument` for tool inputs and outputs.